test/utils/isValidAddress.ts

```diff
@@ -0,0 +1,25 @@
+import { assert } from 'chai'
+
+import { isValidAddress } from '../../src'
```

Comment:
For browser builds this should import from 'xrpl-local'

Reply:
This test isn't run on the browser - it's a unit test, not an integration test.